### ISSUE TITLE
Xxx streams (eu) next missing

### ISSUE THREAD

**Issue by wesleycars** [dobbelina/repository.dobbelina#75]
I recently went onto xxx streams (eu) and  noticed the next button is gone on main page. Before it was always there and I also updated to the 1.1.90 to see if that fixed the bug that caused it to disappear but it didn’t come back.

**Comment by DJMix-I-Can** [comment]
Change line 46 to nextp = re.compile('<a class="nextpostslink.*?href="(.+?)">', re.DOTALL | re.IGNORECASE).findall(html)
[xxxstreams.zip](https://github.com/dobbelina/repository.dobbelina/files/3449400/xxxstreams.zip)

**Comment by wesleycars** [comment]
I forgot to mention that this on kodi on a firestick, so how would I do this?

**Comment by DJMix-I-Can** [comment]
When I had the fire stick, I used the Apps2Fire app. I would download zips on my phone and transfer it with the app via WiFi.  It would transfer it to the sticks download folder.  For pics or any file that isn't a zip, I would use ES Explorer to navigate through the sticks folders. I stopped using ES Explorer because it started throwing alot ads. Now I use FX Explorer.  I still use Apps2Fire & FX Explorer with my Sheild. Unfortunately, Apps2Fire is only for Android.
